content_scripts/VirtualCamera.js

```diff
@@ -131,15 +139,40 @@ import * as THREE from '../../thirdPartyCode/three/three.module.js';
                 }
             }.bind(this));
 
+            // when in normal mode, right click to add a green focus cube to the scene
+            const setFocusTargetCube = (event) => {
+                if (this.isFlying) return;
+                // conform to spatial cursor mousemove event pageX and pageY
+                if (event.button === 2) {
+                    let worldIntersectPoint = realityEditor.spatialCursor.getRaycastCoordinates(event.pageX, event.pageY).point;
```

Comment:
When I run the code, I get an error that `realityEditor.spatialCursor.getRaycastCoordinates` is not a function. Can you make a corresponding PR in the userinterface branch where you export the getRaycastCoordinates function? (I got it working by adding that line to my code)